modules/log6x-log-collector-syslog-server.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/log_collection_forwarding/cluster-logging-collector.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="log-collector-syslog-server_{context}"]
+= Configuring the collector to listen for connections as a syslog server
+
+You can configure your log collector to collect journal format infrastructure logs by specifying `syslog` as a receiver input in the `ClusterLogForwarder` custom resource (CR).
+
+:feature-name: Syslog receiver input
+include::snippets/logging-http-sys-input-support.adoc[]
+
+
+.Prerequisites
+
+* You have administrator permissions.
+* You have installed the {oc-first}.
+* You have installed the {clo}.
+* You have created a `ClusterLogForwarder` CR.
+
+.Procedure
+
+. Modify the `ClusterLogForwarder` CR to add configuration for the `syslog` receiver input:
++
+.Example `ClusterLogForwarder` CR
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+# ...
+spec:
+  inputs:
+    - name: syslog-receiver # <1>
+      type: receiver
+      receiver:
+        type: syslog # <2>
+        port: 10514 # <3>
+        syslog:
+          format: kubeAPIAudit   
+  pipelines: # <4>
+    - name: syslog-pipeline
+      inputRefs:
+        - syslog-receiver
+# ...
+----
+<1> Specify a name for your input receiver.
+<2> Specify the input receiver type as `syslog`.
+<3> Optional: Specify the port that the input receiver listens on. This must be a value between `1024` and `65535`. The default value is `10514` if this is not specified.
+<4> Configure a pipeline for your input receiver.
+
+. Apply the changes to the `ClusterLogForwarder` CR by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <filename>.yaml
+----
+
```

Comment:
Can you add the expected result?  Once CLF is created, the collector listen on service <clf-name>-<inputs.[x].name>
```
$ oc get svc
NAME                        TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)            AGE
collector                   ClusterIP   172.30.85.239    <none>        24231/TCP          33m
collector-syslog-receiver   ClusterIP   172.30.216.142   <none>        10514/TCP          2m20s
```